scrapers/pjgirls.yml

```
@@ -0,0 +1,37 @@
+name: pjgirls
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - pjgirls.com
```

Make sure we use the videos and not the pictures
```suggestion
      - pjgirls.com/en/video
```